reggie/configs/data/pennsylvania.yaml

```diff
@@ -235,3 +355,157 @@ county_names:
   - Westmoreland
   - Wyoming
   - York
+dtypes:
```

Comment:
take these out if possible, since it's all str anyway and you can just derive the list from ordered_columns